go-controller/pkg/util/subnet_annotations.go

```diff
@@ -136,7 +136,14 @@ func parseSubnetAnnotation(nodeAnnotations map[string]string, annotationName str
 }
 
 func NodeSubnetAnnotationChanged(oldNode, newNode *v1.Node) bool {
-	return oldNode.Annotations[ovnNodeSubnets] != newNode.Annotations[ovnNodeSubnets]
+	oldNodeSubnets, oldOk := oldNode.Annotations[ovnNodeSubnets]
```

Comment:
Please ignore the changes in this function.  I'll revert it.
It was by mistake.